cli-plugins/manager/metadata.go

```diff
@@ -22,4 +22,6 @@ type Metadata struct {
 	ShortDescription string `json:",omitempty"`
 	// URL is a pointer to the plugin's homepage.
 	URL string `json:",omitempty"`
+	// Experimental specifies whether the plugin is experimental
```

Comment:
nit: Add more context? Something like "and then won't be displayed unless the CLI has experimental enabled" ?